projects/pastanaga-angular/src/lib/styles/_cssvariables.scss

```diff
@@ -1,31 +1,31 @@
 @import "variables";
 
 :root {
-    --color-neutral-primary-default: #{$color-neutral-primary-default};
-    --color-neutral-primary-lighter: #{$color-neutral-primary-lighter};
-    --color-neutral-primary-lightest: #{$color-neutral-primary-lightest};
-    --color-neutral-secondary-default: #{$color-neutral-secondary-default};
-    --color-neutral-secondary-light: #{$color-neutral-secondary-light};
-    --color-neutral-secondary-lighter: #{$color-neutral-secondary-lighter};
-    --color-neutral-secondary-lightest: #{$color-neutral-secondary-lightest};
-    --color-accent-primary-darker: #{$color-accent-primary-darker};
-    --color-accent-primary-dark: #{$color-accent-primary-dark};
-    --color-accent-primary-default: #{$color-accent-primary-default};
-    --color-accent-primary-light: #{$color-accent-primary-light};
-    --color-accent-primary-lighter: #{$color-accent-primary-lighter};
-    --color-accent-primary-lightest: #{$color-accent-primary-lightest};
-    --color-accent-secondary-darker: #{$color-accent-secondary-darker};
-    --color-accent-secondary-dark: #{$color-accent-secondary-dark};
-    --color-accent-secondary-default: #{$color-accent-secondary-default};
-    --color-accent-secondary-light: #{$color-accent-secondary-light};
-    --color-accent-secondary-lighter: #{$color-accent-secondary-lighter};
-    --color-accent-secondary-lightest: #{$color-accent-secondary-lightest};
-    --color-accent-tertiary-darker: #{$color-accent-tertiary-darker};
-    --color-accent-tertiary-dark: #{$color-accent-tertiary-dark};
-    --color-accent-tertiary-default: #{$color-accent-tertiary-default};
-    --color-accent-tertiary-light: #{$color-accent-tertiary-light};
-    --color-accent-tertiary-lighter: #{$color-accent-tertiary-lighter};
-    --color-accent-tertiary-lightest: #{$color-accent-tertiary-lightest};
+    --color-neutral-primary-default: #{$color-black};
```

Comment:
ok, the idea here is to always have the same names for scss variables and css variables
But no big deal, I can update myself it later if you want.

Reply:
I was not aware of that file. I can do the change. It's the one used for your personal project?